arlo-client/src/types.ts

```diff
@@ -88,6 +88,7 @@ export interface IAuditBoard {
   jurisdictionName: string
   roundId: string
   members: IAuditBoardMember[]
+  signedOffAt: string | null
```

Comment:
should this be instead `signedOffAt?: string` so we are not mixing different approaches to nulls?

Reply:
`signedOffAt?: string` is equivalent to `signedOffAt: string | null | undefined`, which is a slightly different meaning (the field might not be on the object). Not the most important distinction in JS though...